### DISCUSSION TITLE
Can't add environment variables in vercel

### DISCUSSION THREAD

**Discussion by ekat-br** [vercel/next.js#52659] [Help]
### Summary

Hello,
I used vercel for the first time today. I was able to deploy my project. However, I did not have the option to add environment variables.   It is not possible for me to enter anything in the input field in the settings of my deployed project. Unfortunately, I have not found any clues as to why this is so. I have deleted the project and deployed again. When importing the Github repository, I was able to add the desired environment variable, but I could not add any other environment variables in the settings. I appreciate any help to solve the problem. 🙏

**Comment by hyjoong** [comment]
Check the following route


https://vercel.com/project/your-project-name/settings/environment-variables

![image](https://github.com/vercel/next.js/assets/70426440/85bd5b43-c796-4a3c-9ee4-56fa769fc7a0)

**Comment by ekat-br** [comment]
Update: I just looked again in vercel and now I can enter Environment Variables normally again.
